app/FAIRshakeWeb/view.py

```diff
@@ -43,8 +55,129 @@ def register():
+    except:
+      pass
+    assessment_count[resource.id] = len(assessment.get(object=resource.id)) # all assessments from all users for all rubrics
+    aggregate_scores[resource.id] = score.get(id=resource.id,kind='text/html') # scores is an array of html with tooltip script
+  current_user_assessments=['1'] ###
+  return render_template('project_resources.html',
+    project=repository.get(id=project,limit=1), ###
+    resources=resources,
+    aggregate_scores=aggregate_scores,
+    assessment_count=assessment_count,
+    current_user_assessments=current_user_assessments,
+    current_user={},
+  )
+
+@app.route('/project/<int:project>/my_evaluations', methods=['GET'])
+def my_evaluations(repository: RepositoryAPI, assessment: AssessmentAPI, score: ScoreAPI, project):
+  project_resources=repository.get(tag=project)
+  current_user_assessed_resources = []
+  assessment_count = {}
+  aggregate_scores = {}
+  current_user_scores = {}
+  for resource in project_resources:
+    try:
+      assessment.get(object=resource.id, user=current_user['id'])
+      current_user_assessed_resources.append(resource)
+    except:
+      pass
+    assessment_count[resource.id] = len(assessment.get(object=resource.id)) # all assessments from all users for all rubrics
+    aggregate_scores[resource.id] = score.get(id=resource.id,kind='text/html') # scores is an array of html with tooltip script
+    current_user_scores[resource.id] = score.get(id=resource.id,user=current_user['id'],kind='text/html')
+  current_user_assessed_resources=[repository.get(limit=1)] ###
+  return render_template('project_evaluated_resources.html',
+    project=repository.get(id=project,limit=1), ###
+    aggregate_scores=aggregate_scores,
+    current_user_scores=current_user_scores,
+    assessment_count=assessment_count,
+    current_user_assessed_resources=current_user_assessed_resources,
     current_user={},
   )
+
+@app.route('/new_evaluation', methods=['GET','POST'])
+def new_evaluation(repository: RepositoryAPI, rubric: RubricAPI, assessment: AssessmentAPI):
+  if request.method == 'GET':
+    resource_id = request.args.get('id')
+    return render_template('new_evaluation.html',
+      resource=repository.get(id=resource_id,limit=1),
+      rubrics=rubric.get(id=resource_id),
+      rubric_ids=[rubric.id for rubric in rubric.get(id=resource_id)],
+      current_user={},
+    )
+  else:
+    resource_id=request.form.get('resource_id')
+    rubric_ids=json.loads(request.form.get('rubric_ids'))
+    rubrics=[rubric.get(id=rubric_id)[0] for rubric_id in rubric_ids]
+    for rubric_index, rubric in enumerate(rubrics):
+      criteria = [] # evaluations
+      for criterion_index, criterion in enumerate(rubric.criteria): # questions
+        criteria.append(
+          {
+            'name': criterion.name,
+            'value': request.form.get('q-{:d}.{:d}'.format(rubric_index,criterion_index))
+          }
+        )
+      assessmentModel = {
```

Comment:
assessmentModel should use the actual model defined in `interfaces/assessment.py`. It can be used like `AssessmentModel(object=resource_id, ...)`